diff --git a/retroarch.c b/retroarch.c
index c8ed7369e8..55f7fe4eb9 100644
--- a/retroarch.c
+++ b/retroarch.c
@@ -3520,7 +3520,8 @@ int generic_menu_entry_action(
                   entry->label, entry->type, i, entry->entry_idx);
          break;
       case MENU_ACTION_SEARCH:
-         menu_input_dialog_start_search();
+         // disable search action
+         // menu_input_dialog_start_search();
          break;
       case MENU_ACTION_SCAN:
          if (cbs && cbs->action_scan)
-- 
2.25.1

